src/controllers/contacts.ts

```diff
@@ -210,10 +212,22 @@ export const generateToken = async (req, res) => {
     if (isProxy()) {
       tribes.subscribe(`${pubkey}/#`, network.receiveMqttMessage) // add MQTT subsription
     }
+    //  create transport token and send back to client
+    // save private key and send public key
+    const transportTokenKeys: { [k: string]: string } = await rsa.genKeys()
+    // TODO: save transport private key
     owner.update({ authToken: hash })
+    // Send transport pubkey
+    fs.writeFileSync('transportPrivate.pem', transportTokenKeys.private)
+    success(res, {
+      id: (owner && owner.id) || 0,
+      transportToken: transportTokenKeys.public,
```

Comment:
I'm sending back the transport token public key as a response when a user successfully generates their `x-user-token`